README.rst

```diff
@@ -17,7 +17,3 @@ Kinto
 
 Kinto is a server allowing you to store and synchronize arbitrary data,
 attached to your Firefox account.
-
-* `Online documentation <http://kinto.readthedocs.org/en/latest/>`_
```

Comment:
Yes I would rather remove the inclusion of the README from the documentation if this bothers you.

Also it is a common practice to see the documentation as a read-more of the readme and people may display the documentation on another domain, in that case having a link to the "official" documentation is not a problem.